cmd/terramate/e2etests/run_cloud_config_test.go

```diff
@@ -118,45 +113,30 @@ func TestCloudConfig(t *testing.T) {
 		tc := tc
 		t.Run(tc.name, func(t *testing.T) {
 			t.Parallel()
-			router := testserver.RouterWith(map[string]bool{
-				cloud.UsersPath:       true,
-				cloud.MembershipsPath: false,
-				cloud.DeploymentsPath: true,
-				cloud.DriftsPath:      true,
+			store, err := cloudstore.LoadDatastore(testserverJSONFile)
+			assert.NoError(t, err)
+
+			store.UpsertOrg(cloudstore.Org{
```

Comment:
TODO: Should we handle errors? 🤔

Reply:
the UpsertOrg() method does not fail (in-memory store).